scripts/temp_convertElectrodes_step2.m

```diff
@@ -0,0 +1,41 @@
+
+%%
+
+working_dir = fullfile('/Fridge','users','jaap','ccep','dataBIDS');
```

Comment:
Add a comment above about "this is where the original identified electrode locations are stored in a matrix called 'elecmatrix that has size electrodes * x,y,z'"

Reply:
added